packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -18,6 +18,7 @@ export class OfficeMockObject {
    * Adds a function to OfficeMockObject. The function can be accessed by simply calling `this.methodName`
    * @param methodName Method name of the function to be added
    * @param methodBody Function to be added to the object. A blank function will be added if no argument is provided (Optional)
+   * @deprecated Use setProperty instead
```

Comment:
Use setProperty instead of addMockFunction?

Reply:
Yes, the `setProperty` is simply now a check that if it is a function it behaves as the `addMockFunction` behaved and if it is not, it will behave as the `setMock` did.